borgmatic/commands/borgmatic.py

```diff
@@ -727,6 +742,7 @@ def main():  # pragma: no cover
         sys.exit(0)
 
     config_filenames = tuple(collect.collect_config_filenames(global_arguments.config_paths))
+    global_arguments.used_config_paths = list(config_filenames)
```

Comment:
I feel a little weird about this monkey-patching, but not weird enough to raise an objection over it. I guess we'll see how it goes!